### PR TITLE
[FIX] core: wsgi application entrypoint moved

### PR DESCRIPTION
**Pull request by Julien00859**
The wsgi application entrypoint moved during the httpocalypse. Some clients don't use the odoo builtin wsgi server and have troubles upgrading from 15.0 to 16.0 because the `odoo.service.wsgi_server` module doesn't exist anymore.

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by Gorash — APPROVED** [review]
Very good (good fix for our forgetfulness)

**Comment by Julien00859** [comment]
@robodoo r+

**Comment by Julien00859** [comment]
@robodoo r-

File: odoo/service/wsgi_server.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import warnings
+
+
+def application(environ, start_response):
+    import odoo.http
+
+    warnings.warn("The WSGI application entrypoint moved from "
+                  "odoo.service.wsgi_server.application to odoo.http.root "
+                  "in 15.3.",
+                  DeprecationWarning, stacklevel=1)
```
Comment (Julien00859) [inline odoo/service/wsgi_server.py:10]: `stacklevel=2` would had log the line where the application is invoked in the WSGI server. That's not what we want here.

**Review by Gorash — APPROVED** [review]
Good for me, the details have been discussed.

**Comment by Julien00859** [comment]
@robodoo r+

**Comment by Julien00859** [comment]
@robodoo r+

**Pull request closed without merging**